geode-connectors/src/distributedTest/java/org/apache/geode/connectors/jdbc/internal/cli/JdbcClusterConfigDistributedTest.java

```diff
@@ -51,6 +51,9 @@ public static void beforeClass() {
   @Test
   public void recreateCacheFromClusterConfig() throws Exception {
     gfsh.connectAndVerify(locator);
+
+    gfsh.executeAndAssertThat("create region --name=regionName --type=REPLICATE").statusIsSuccess();
```

Comment:
This region is a replicated region. And the DUnit tests above all use replicated region. Just wonder if there is some test coverage for partitioned region.

Reply:
good idea